### PR TITLE
Initial commit.

### PR DESCRIPTION
**Pull request by tonybaltovski**

### CODE REVIEW

**Comment by roni-kreinin** [comment]
Are these meant to be generic messages, or Jackal messages?

**Comment by tonybaltovski** [comment]
Generic messages.

**Comment by roni-kreinin** [comment]
Should we use these? https://github.com/clearpathrobotics/cpr_common_core/tree/foxy-devel/cpr_common_core_msgs

File: msg/Drive.msg [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# This message represents a low-level motor command to Jackal.
+
+# Command units dependent on the value of this field
+int8 MODE_VELOCITY=0   # velocity command (rad/s of wheels)
+int8 MODE_PWM=1        # proportion of full voltage command [-1.0..1.0]
+int8 MODE_EFFORT=2     # torque command (Nm)
```
[the remark this answers is not in the document]
Comment (tonybaltovski) [reply msg/Drive.msg]: Removed.

**Pull request merged**